### ISSUE TITLE
localhost account contains 0.0001 ETH

### ISSUE THREAD

**Issue by tevis-zorg** [NomicFoundation/hardhat#3927]
### Version of Hardhat

2.14.0

### What happened?

in hardhat node account number 13, i discovered it is filled with the amount of 0.0001 ETH but. while i try to visit the metamask portofolio.

### Minimal reproduction steps

1. npx hardhat node,
2. adding network with my localhost,
3. enters the private from my local node which provided by hardhat

### Search terms

instruction of creating local coin on hardhat.

**Comment by fvictorio** [comment]
@tevis-zorg I'm not sure I understand what this issue is about, can you expand on what's the problem here?

**Comment by tevis-zorg** [comment]
> @tevis-zorg I'm not sure I understand what this issue is about, can you expand on what's the problem here?

sorry to late respond. i'm already update the issue with the screenshot now.
hope it can makes you understand

**Comment by tevis-zorg** [comment]
> I'm still not quite sure what you mean, but apparently the 13th account of hardhat has that balance in Mainnet (https://etherscan.io/address/0x1CBd3b2770909D4e10f157cABC84C7264073C9Ec), so maybe that's what you are seeing?
> 
> If that's your question, then what happened is that probably someone accidentally sent some eth to that account. Since these accounts have well-known private keys, there are bots that immedately steal their funds. In this case there was some leftover for some reason.
> 
> tl;dr, that account will have 10000 ETH in the local development network, but it shows some value in mainnet because of that.

i see. ok, thanks for your help victorio.
